### PR TITLE
Load: Skip the file failed to load instead of stopping loading the whole directory

### PR DESCRIPTION
**Pull request by MiniSho**
## Description
As title mentions.

image

self-reviewed: ✅

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java [hunk 1]
```diff
@@ -103,6 +103,11 @@ public List<WritePlanNode> splitByPartition(Analysis analysis) {
               statement.isDeleteAfterLoad(),
               statement.getWritePointCount(i)));
     }
+
+    for (int i = 0; i < statement.getFailedTsFiles().size(); i++) {
+      res.add(
+          new LoadSingleTsFileNode(getPlanNodeId(), statement.getFailedTsFiles().get(i), false));
```
Comment (SzyWilliam) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java:109]: Why not use `for (File failedTsFile: statement.getFailedTsFiles())` here?

**Review by SzyWilliam — APPROVED** [review]
+1 what a bravo patch! Just some minor issues, bro m3?

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/exception/LoadFailedException.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    final int failedNodeSize = failedNodes.size();
+    for (LoadSingleTsFileNode node : failedNodes) {
+      fileNames.append(node.getTsFile().getName()).append(", ");
+    }
+
+    if (fileNames.length() > 0) {
+      fileNames.setLength(fileNames.length() - 2);
+    }
+
+    return "Failed to load " + failedNodeSize + " files: " + fileNames;
```
Comment (Caideyipi) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/exception/LoadFailedException.java:45]: Use String.join() here may be more graceful.

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/analyze/LoadTsfileAnalyzer.java [hunk 3]
```diff
@@ -170,25 +171,27 @@ public Analysis analyzeFileByFile() {
...
-                "Parse file %s to resource error, because %s", tsFile.getPath(), e.getMessage()));
+        loadTsFileStatement.addFailedTsFile(tsFile);
       }
     }
 
-    try {
-      schemaAutoCreatorAndVerifier.flush();
-    } catch (AuthException e) {
-      return createFailAnalysisForAuthException(e);
+    LOGGER.info(
```
Comment (Caideyipi) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/analyze/LoadTsfileAnalyzer.java:188]: "warn" maybe better?

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java (hunk 1 above)
Comment (Caideyipi) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java:109]: I think stream maybe better :-)

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/scheduler/load/LoadTsFileScheduler.java [hunk 4]
```diff
@@ -189,23 +194,30 @@ public void start() {
...
               node.getTsFileResource().getTsFilePath(),
               e);
         }
       }
-      if (isLoadSuccess) {
+
+      if (!failedTsFileNodes.isEmpty()) {
+        LOGGER.warn("Load - Dispatch Stage: {} TsFiles failed to load.", failedTsFileNodes.size());
+        for (LoadSingleTsFileNode node : failedTsFileNodes) {
+          LOGGER.warn(
```
Comment (Caideyipi) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/scheduler/load/LoadTsFileScheduler.java:216]: Maybe the List<nodes> can be displayed directly?

**Review by Caideyipi — APPROVED** [review]

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java (hunk 1 above)
Comment (MiniSho) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadTsFileNode.java:109]: thanks homie, I'll take your advice :)

**Pull request closed without merging**